apps/viewer/init.js

```diff
@@ -553,6 +553,46 @@ async function initUIcomponents() {
       tour.start(true);
     },
   });
+
+  // Additional Links handler
+  var headers = {
+    'Access-Control-Allow-Origin': '*',
+  };
+
+  function additionalLinksHandler(url, openInNewTab) {
+    if (openInNewTab === true) {
+      window.open(url, '_blank').focus();
+    } else {
+      window.location.href = url;
+    }
+  }
+
+  var additionalLinksFetchResponse = await fetch('http://localhost:4010/additional_links.json', {headers: headers});
+  // Handle error
+  if (!additionalLinksFetchResponse.ok) {
+    var message = `Error ${additionalLinksFetchResponse.status}: File not found or file not in correct format`;
+    alert(message);
+    throw new Error(message);
```

Comment:
I think that should the file not exist, the expected behavior should be equivalent to just adding no additional links.

Reply:
Alright I'll do that

Reply:
@birm Should I remove the alert then? And just keep it as throwing an error?

Reply:
More important is the idea that a bad/missing file should not stop the code from executing.

Reply:
Yes, I've done that. Also removed the alerts